timesync/peersync/sync.go

```diff
@@ -23,7 +23,7 @@ const (
 
 var (
 	// ErrPeersNotSynced returned if system clock is out of sync with peers clock for configured period of time.
-	ErrPeersNotSynced = errors.New("timesync: peers are not time synced")
+	ErrPeersNotSynced = errors.New("timesync: peers are not time synced, make sure your system clock is accurate")
```

Comment:
you could also mention how to disable timesync if you are confident that your clock is accurate. below is a configuration to disable it

```json
{
    "time": {
        "peersync": {
            "disable": true
        }
    },
```

Reply:
I think i'll add it to smesher-guide